src/app/manager-dashboard/reports/myplanet-table.component.ts

```diff
@@ -39,6 +42,17 @@ export class MyPlanetTableComponent implements OnInit, OnChanges, AfterViewInit
 
   ngOnChanges() {
     this.myPlanets.data = this.data;
+    if (this.usage !== undefined) {
+      let total;
+      this.usage.map(data => {
+        data.max.usages.map((time) => {
```

Comment:
This is not a good use for `Array.map`.  You are totaling up values based on the array provided, and `Array.reduce` is much better for that.  Then you can do `const total = data.max.usages.reduce(...)`.